### PR TITLE
Added alignitemscenter class for vertical alignment

### PR DESCRIPTION
**Pull request by yratof**
Gallery assumes that all images are created equally. Added class to vert align while the admin kitchensync option is still pending

### CODE REVIEW

File: packages/block-library/src/gallery/style.scss [hunk 1]
```diff
@@ -147,4 +147,11 @@
 			justify-content: center;
 		}
 	}
+
+	// Align the gallery images vertically if images vary in height.
+	&.alignitemscenter {
```
Comment (youknowriad) [inline packages/block-library/src/gallery/style.scss]: Where is this class coming from. I can't seem to find it in the repo?

**Comment by richtabor** [comment]
@yratof I'm not quite following what `align-items: center;` would achieve here. I think it'd be best to create a relevant issue (or link to an existing one if one exists) with screenshots showing the issue and how `align-items: center;` would be an appropriate fix. 

Then we would need to decide how the class would get applied (which depends on the issue and how appropriate the fix is).

Thanks!

**Comment by joyously** [comment]
If you add a class, have the name tell what it's for. (maybe put `vertical` in there)

**Comment by karmatosed** [comment]
For now, I am going to close this as seems it's been sitting waiting for feedback for a while. It can always be reopened if needed.

**Pull request closed without merging**